R/calcFEUEefficiencies.R

```diff
@@ -67,8 +67,23 @@ calcFEUEefficiencies <- function(gasBioEquality = TRUE) {
                                       where = "mredgebuildings")
 
 
+  # Cooling COP boundaries
+  coolingBounds <- toolGetMapping("coolingEfficiencyBoundaries.csv",
+                                  type = "sectoral",
+                                  where = "mredgebuildings")
+
+
+  # PARAMETERS -----------------------------------------------------------------
+
+  # Weight given to log(gdppop) for space_cooling.elec input variable
+  #   -> x = log(gdppop) * weight + period * (1 - weight)
+  gdppopWeight <- 0.95
+
+
   # PROCESS DATA ---------------------------------------------------------------
 
+  coolingBounds <- setNames(coolingBounds$value, coolingBounds$variable)
```

Comment:
Slightly more compact:
```suggestion
  coolingBounds <- pull(coolingBounds, "value", "variable")
```